testnet/stacks-node/src/tests/signer.rs

```diff
@@ -103,30 +104,21 @@ impl SignerTest {
             Some(Duration::from_millis(128)), // Timeout defaults to 5 seconds. Let's override it to 128 milliseconds.
         );
 
-        let mut running_signers = vec![];
-        let mut _signer_cmd_senders = vec![];
+        let mut running_signers = HashMap::new();
+        let mut _signer_cmd_senders = HashMap::new();
         // Spawn all the signers first to listen to the coordinator request for dkg
```

Comment:
Nit this comment is no longer relevant and can be either removed or changed to "// Spawn all signers before the node to ensure their listening ports are open for the node event observer to bind to"